### PR TITLE
Removed Project_Readme from CSharp_Web

### PR DESCRIPTION
**Pull request by seancpeters**
Added the package fallback target to CSharp_Web

### CODE REVIEW

**Comment by seancpeters** [comment]
Good catch, that’s a mistake from my copy-paste ☹… fixing

**Comment by seancpeters** [comment]
The $if$ is in an odd spot to keep spacing consistent with the VS version of the template.

File: src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj [hunk 1]
```diff
@@ -5,7 +5,11 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp1.0</TargetFramework>
     <PreserveCompilationContext>true</PreserveCompilationContext>
-  </PropertyGroup>
+  </PropertyGroup>$if$ ($context$ == WebCore)
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj]: We do not handle these in the CLI today. I don't understand what this is doing in the CLI template.

**Comment by seancpeters** [comment]
Sorry about that. I misunderstood. I've removed the if-else entirely.

**Comment by livarcocc** [comment]
@seancpeters All right. We will have those capabilities soon I hope, but not yet.

**Comment by livarcocc** [comment]
Though, why are we doing this now? Do we need approval for this?

**Comment by piotrpMSFT** [comment]
This is a pre-req for the web sdk update we're doing later tonight/tomorrow morning.

**Pull request merged**